Simplenote/Classes/SPSettingsViewController.m

```diff
@@ -257,6 +281,12 @@ - (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInte
         case SPOptionsViewSectionsAppearance:
             return NSLocalizedString(@"Appearance", nil);
 
+        case SPOptionsViewSectionsSustainer:
+            if ([self isActiveSustainer]) {
+                return NSLocalizedString(@"Sustainer Thank You", nil);
```

Comment:
We maaay need to check with someone this copy. Perhaps we could poke either Sly or Dave?